content/_index.en.md

```diff
@@ -105,14 +105,22 @@ draft: false
 {{< /persons >}}
 
 {{< roadmap "Roadmap" >}}
-  {{< roadmap_item "left" "Q1 2025" "Fast search, N3 accounts in NeoFS, more meta" >}}
+  {{< roadmap_item "right" "Q2 2025" "N3 accounts in NeoFS, new token model" >}}
     <ul>
-      <li>More performant and capable search API</li>
       <li>Full support for N3 accounts in NeoFS</li>
-      <li>New metadata handling subsystem prototype</li>
+      <li>Token model rework to support more usage scenarios</li>
       <li>Snapshot-based blockchain synchronization</li>
     </ul>
   {{< /roadmap_item >}}
+  {{< roadmap_item "left" "Q1 2025" "Fast search, more meta" >}}
+    <ul>
+      <li>More performant and capable search API</li>
+      <li><a href="https://archive.fs.neo.org/">archive.fs.neo.org</a></li>
```

Comment:
`target="_blank"`?

Reply:
Questionable to me. I'd rather have it open in the same window, it's up to user to decide if he wants an additional tab.